SwiftGit2/SwiftGit2.m

```diff
@@ -34,3 +34,21 @@ git_checkout_options SG2CheckoutOptions(SG2CheckoutProgressBlock progress) {
 	
 	return result;
 }
+
+git_clone_options SG2CloneOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_clone_options result = GIT_CLONE_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
+}
+
+git_fetch_options SG2FetchOptions() {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-field-initializers"
+	git_fetch_options result = GIT_FETCH_OPTIONS_INIT;
+#pragma clang diagnostic pop
+
+	return result;
```

Comment:
It would be nice if these could follow the example of `SG2CheckoutOptions` and do all the configuring inside this function. Ideally, `cloneFromURL` wouldn't need to do any manual configuration.